tme/handlers.go

```diff
@@ -0,0 +1,193 @@
+package tme
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"strconv"
+
+	"strings"
+
+	"github.com/Financial-Times/go-fthealth/v1a"
+	"github.com/Financial-Times/service-status-go/gtg"
+	"github.com/Financial-Times/transactionid-utils-go"
+	log "github.com/Sirupsen/logrus"
+	"github.com/gorilla/handlers"
+	"github.com/gorilla/mux"
+	"github.com/pkg/errors"
+)
+
+type Handler struct {
+	service Service
+}
+
+func NewHandler(service Service) *Handler {
+	return &Handler{
+		service: service,
+	}
+}
+
+func (th *Handler) HandleGetFullTaxonomy(resp http.ResponseWriter, req *http.Request) {
+	vars := mux.Vars(req)
+	t := vars["type"]
```

Comment:
I'd replace literal "type" with a constant and would be more explicit what it refers to.  In addition in the API doc this path var is referred to as "taxonomy". Also your method refers to "Taxonomy".